src/main/java/uk/gov/justice/digital/delius/controller/secure/CustodyResource.java

```diff
@@ -56,5 +57,19 @@ public Custody updateCustodyBookingNumber(final @PathVariable String nomsNumber,
         return custodyService.updateCustodyBookingNumber(nomsNumber, updateCustodyBookingNumber);
     }
 
+    @RequestMapping(value = "offenders/crn/{crn}/nomsNumber", method = RequestMethod.PUT, consumes = "application/json")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "The offender record was updated", response = IDs.class),
+            @ApiResponse(code = 401, message = "Request is missing Authorization header (no JWT)"),
+            @ApiResponse(code = 404, message = "The requested offender was not found")
+    })
+    @ApiOperation(value = "Updates the offender record with the NOMS number in UpdateOffenderNomsNumber")
+    public IDs updateOffenderNomsNumber(final @PathVariable String crn,
+                                              final @RequestBody @Valid UpdateOffenderNomsNumber updateOffenderNomsNumber) {
```

Comment:
does the `@Valid` annotation work without a `Validated` annotation? Can't see a test to confirm my suspicions...

Reply:
actually just seen the 400 example below so sounds like it does, but wondering now about the `NotBlank` i.e. will spaces for the noms number work too?

Reply:
I'll add a test to check